Sources/StreamChatUI/ChatChannel/ChatChannelVC.swift

```diff
@@ -342,7 +342,7 @@ open class ChatChannelVC: _ViewController,
         willDisplayMessageAt indexPath: IndexPath
     ) {
         let message = chatMessageListVC(vc, messageAt: indexPath)
-        if message?.id == firstUnreadMessageId {
+        if message?.id == firstUnreadMessageId || firstUnreadMessageId == nil {
```

Comment:
If there is no firstUnreadMessageId, then it means we have seen all messages as well